src/test/java/api/support/fixtures/TemplateContextMatchers.java

```diff
@@ -66,7 +64,7 @@ private TemplateContextMatchers() {
     tokenMatchers.put("item.callNumber", is(callNumber));
     tokenMatchers.put("item.callNumberPrefix", is(callNumberPrefix));
     tokenMatchers.put("item.callNumberSuffix", is(callNumberSuffix));
-    tokenMatchers.put("item.copy", is(copyNumbers));
+    tokenMatchers.put("item.copy", is(copyNumber));
```

Comment:
Does this impact the template definitions for patron notices?

Reply:
No, when we had several values for `item.copyNumbers` they were built as a single string, separated by a semicolon.

I think it will work fine for a single value as well.